PyTest Mocks: Undefined name 'pytest_funcarg__XYZ'

@hornc In this commit https://github.com/internetarchive/openlibrary/commit/009a42b20ef86bfae947f23844528d74b3197366 you did some magic around PyTest and we would like to see if you can that magic again around these two _undefined names_.  It would help us on #1684 

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/mocks/mock_ol.py#L49-L50

python -m flake8 ./openlibrary/mocks --select=F821 --show-source
```
./openlibrary/mocks/mock_ol.py:49:28: F821 undefined name 'pytest_funcarg__monkeypatch'
        self.monkeypatch = pytest_funcarg__monkeypatch(request)
                           ^
./openlibrary/mocks/mock_ol.py:50:21: F821 undefined name 'pytest_funcarg__mock_site'
        self.site = pytest_funcarg__mock_site(request)
                    ^
```

--- a/Makefile
+++ b/Makefile
@@ -104,7 +104,7 @@ lint:
 	$(PYTHON) -m flake8 . --exit-zero --count --exclude=./.*,scripts/20*,vendor/*  --select=F821 --show-source --statistics
 ifndef CONTINUOUS_INTEGRATION
 	# exit-zero treats all errors as warnings, only run this in local dev while fixing issue, not CI as it will never fail.
-	$(PYTHON) -m flake8 . --count --exclude=./.*,scripts/20*,vendor* --exit-zero --max-complexity=10 --max-line-length=$(GITHUB_EDITOR_WIDTH) --statistics
+	$(PYTHON) -m flake8 . --count --exclude=./.*,scripts/20*,vendor*,node_modules/* --exit-zero --max-complexity=10 --max-line-length=$(GITHUB_EDITOR_WIDTH) --statistics
 endif
 
 test:

--- a/openlibrary/mocks/mock_ol.py
+++ b/openlibrary/mocks/mock_ol.py
@@ -43,11 +43,12 @@ class OLBrowser(web.AppBrowser):
 class OL:
     """Mock OL object for all tests.
     """
-    def __init__(self, request):
+    @pytest.fixture
+    def __init__(self, request, monkeypatch):
         self.request = request
 
-        self.monkeypatch = pytest_funcarg__monkeypatch(request)
-        self.site = pytest_funcarg__mock_site(request)
+        self.monkeypatch = monkeypatch(request)
+        self.site = mock_site(request)
 
         self.monkeypatch.setattr(ol_infobase, "init_plugin", lambda: None)
 

